package.json

```diff
@@ -59,8 +59,10 @@
     "eslint": "cross-env TIMING=1 eslint --ext .js,.ts build/types src test",
     "eslint:fix": "cross-env TIMING=1 eslint --ext .js,.ts --fix build/types src test",
     "autocannon": "autocannon",
-    "mochatest": "cross-env TS_NODE_FILES=true nyc mocha test/**/*Test.ts",
-    "mochatest:no-parallel": "cross-env TS_NODE_FILES=true nyc mocha --no-parallel test/**/*Test.ts",
+    "mochatest": "cross-env TS_NODE_FILES=true mocha --no-parallel test/**/*Test.ts",
+    "mochatest:parallel": "cross-env TS_NODE_FILES=true mocha test/**/*Test.ts",
+    "mochatest:nyc": "cross-env TS_NODE_FILES=true nyc mocha --no-parallel test/**/*Test.ts",
+    "mochatest:nyc-parallel": "cross-env TS_NODE_FILES=true nyc mocha test/**/*Test.ts",
```

Comment:
The default setting must be parallel to avoid time wasting (50% supplemental time without parallel) for developers and GitHub actions. 
And the de facto standard for scripts naming is: `command:cli-options`.
If debug is needed, the explicit no-parallel scripts are here for that reason.

Reply:
The parallel execution triggers a lot of failures in the unit tests on my side.
Putting this option by default is longer but the unit tests are all passing successfully which I prefer then.
And you know what I think about the standard stuff already ;-)

Reply:
> The parallel execution triggers a lot of failures in the unit tests on my side.
> Putting this option by default is longer but the unit tests are all passing successfully which I prefer then.

I think you have an issue with your setup, the parallel mode since its enablement is working well for all developers and in the GitHub action. 
If want to revert it properly, you have to: 
+ set parallel to false in the mocha configuration file, 
+ use the --parallel option on moche in scripts requiring it,
+ run the test script with parallel mode in the GitHub action. 

> And you know what I think about the standard stuff already ;-)

De facto standard are made by developers for developers and here it's just common sense: calling a command with one option but the naming scheme state the contrary ...

Reply:
I think Madhi has also issues on running cars unit tests (random.)
I'm not the only one here, I'm pretty sure.
And I'm having these random issues since a long time, just found the root cause today.